index.js

```diff
@@ -483,18 +483,29 @@
 
   //. ### Classify
 
-  //# type :: Any -> String
+  //# type :: Any -> { namespace :: Maybe String, name :: String, version :: Integer }
```

Comment:
I think awkward is better than incorrect. It seems the root of the problem lies in the fact that `type.parse` doesn't preserve whether the version was given or not.

In the case of [type identifier derivation](https://github.com/fluture-js/concurrify/blob/2db86183a03e5d8a8a3f3d0161c6705ab6b0da2d/index.js#L64-L68), that's actually nice because we don't have to do the whole `+ (o.version ? '@' + o.version : '')` thing.

We could change the output of `type.parse` to use `Nullable Integer`, preserving the absence of a version, but that would mean extra null checks of the kind I showed above. Another idea is to add something like `type.isNative :: Any -> Boolean` or `type.hasVersion :: String -> Boolean`. I'm weighing more towards the former though.